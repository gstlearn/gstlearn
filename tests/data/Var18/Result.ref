Goodness-of-Fit (as a percentage of the Sill) may demonstrate an issue:  5.47
