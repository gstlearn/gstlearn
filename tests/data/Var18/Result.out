
Model characteristics
=====================
Space dimension              = 3
Number of variable(s)        = 2
Number of basic structure(s) = 2
Number of drift function(s)  = 0
Number of drift equation(s)  = 0

Covariance Part
---------------
Cubic
- Sill matrix:
               [,  1]    [,  2]
     [  1,]     0.159     0.169
     [  2,]     0.169     0.319
- Range        =      1.289
Spherical
- Sill matrix:
               [,  1]    [,  2]
     [  1,]     0.594     0.126
     [  2,]     0.126     0.298
- Range        =      4.291
Total Sill
               [,  1]    [,  2]
     [  1,]     0.753     0.295
     [  2,]     0.295     0.616

